# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../argus/drng-hotfix.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./include/argus_client.h.orig	2006-09-08 18:51:08.000000000 +0200
+++ ./include/argus_client.h	2006-12-14 00:27:24.000000000 +0100
@@ -305,7 +305,7 @@
    "vpri",
    "ipid",
    "srng",
-   "erng",
+   "drng",
    "svc",
 };
 
